plotsquared/vanilla-tags.md

```diff
@@ -1,10 +1,10 @@
 # Vanilla Tags
 
-## What are "Tags" ?
+## What are "Tags?"
```

Comment:
why is the question mark inside the quotes?

Reply:
I'm not sure if this English rule applies (since the "quote" isn't quoting someone), but ending-punctuation is applied in quotes and not outside (or at least that's what I've been taught).

Reply:
page 16 of the [University of Oxford style guide](https://www.ox.ac.uk/sites/files/oxford/media_wysiwyg/University%20of%20Oxford%20Style%20Guide.pdf) uses the punctuation outside the quotes for non-quotes (see the 4th example on the page). so i'll do that.